src/main/java/de/dhbw/karlsruhe/domain/services/dialogs/DifficultySelectionDialogService.java

```diff
@@ -32,7 +38,7 @@ private Difficulty successfulSelection() {
           .findFirst()
           .orElse(null);
```

Comment:
Dieser Prozess mit dem Input einlesen und matchen, müsste denke ich auch in den Adapter.